backend/apps/owasp/models/project.py

```diff
@@ -182,6 +218,18 @@ def pull_requests(self):
             "repository",
         )
 
+    @property
+    def pull_requests_count(self) -> int:
+        """Return count of pull requests."""
+        return self.pull_requests.count()
+
+    @property
+    def pull_request_last_created_at(self) -> datetime.datetime | None:
+        """Return last created pull request."""
+        if pull_request := self.pull_requests.order_by("-created_at").first():
+            return pull_request.created_at
+        return None
```

Comment:
Try:
```suggestion
return self.pull_requests.aggregate(Max("created_at"))["created_at__max"]
```
just check it works correct on empty `self.pull_requests`